### ISSUE TITLE
Add a color to visited links in a user's download history

### ISSUE THREAD

**Issue by shanear** [department-of-veterans-affairs/caseflow-efolder#240]

**Comment by lakohl** [comment]
@shanear we can use the US Design Standards color for 'visited' #4c2c92

**Comment by amprokop** [comment]
![image](https://cloud.githubusercontent.com/assets/3781835/19326577/1d18688c-9099-11e6-9160-4a988b068da8.png)

Just talked to @lakohl. When you type "DEMO123" into the input and press Search, you're immediately redirected to `/downloads/{number}`— so if we use the css class `:visited`, all of these links will always show up as visited. 

The intent of this task was to give users a visual cue to tell them which zip files have been downloaded. There may be better ways to achieve this.

**Comment by aroltsch** [comment]
Waiting on @Chingujo to upload design mock.

**Comment by Chingujo** [comment]
I don't think we do this for any of the links? At least in eFolder Express demo. Will see across apps.
